emoji-copy@felipeftn/emojiCategory.js

```diff
@@ -1,13 +1,13 @@
-const Clutter = imports.gi.Clutter;
-const St = imports.gi.St;
-const PopupMenu = imports.ui.popupMenu;
+import St from 'gi://St';
+import Clutter from 'gi://Clutter';
+import * as PopupMenu from 'resource:///org/gnome/shell/ui/popupMenu.js';
 
 /* Import the current extension, mainly because we need to access other files */
-const ExtensionUtils = imports.misc.extensionUtils;
-const Me = ExtensionUtils.getCurrentExtension();
-const SkinTonesBar = Me.imports.emojiOptionsBar.SkinTonesBar;
-const Extension = Me.imports.extension;
-const EmojiButton = Me.imports.emojiButton;
+import { SkinTonesBar } from './emojiOptionsBar.js';
+import { EmojiButton } from './emojiButton.js';
+import * as Extension from './extension.js'; // Can I get SETTINGS with this? (Extension.SETTINGS)?
+
+import { gettext as _ } from 'resource:///org/gnome/shell/extensions/js/extensions.js';
```

Comment:
This line needs to be changed to
```js
import {gettext as _} from 'resource:///org/gnome/shell/extensions/extension.js';
```

Note that not only the `/js` part from the path is removed, but the filename `extension.js` is now also in singular instead of plural.